.github/workflows/publish_pypi.yml

```diff
@@ -22,8 +22,11 @@ jobs:
         with:
           poetry-version: "1.7.1"
 
+      - name: Create step summary
+        run: echo "## Building $(poetry version)" >> $GITHUB_STEP_SUMMARY
+
       - name: Build package
-        run: poetry build
+        run: echo "$(poetry build)" >> $GITHUB_STEP_SUMMARY
```

Comment:
Wouldn't just `poetry build >> $GITHUB_STEP_SUMMARY` work?

Reply:
Yeah, and probably also in one or two other places. Copy-pasted-changed without thinking 🤦